react_demo/vendor/react

```diff
@@ -0,0 +1 @@
+react-0.13.3-build/
```

Comment:
Does not match the path you serve (the symlink is in the wrong folder, should be in `static`).